src/Readline/Hoa/Xcallable.php

```diff
@@ -117,10 +117,10 @@ public function __construct($call, $able = '')
                 }
             } elseif (\is_array($call) && isset($call[0])) {
                 if (!isset($call[1])) {
-                    return $this->__construct($call[0]);
+                    $this->__construct($call[0]);
```

Comment:
the point of the return here (and below) was to prevent execution from continuing. if we're not going to return the result of `__construct` (which is harmless, but ¯\\\_(ツ)\_/¯) then we need a bare `return` on the next line.